hles/extraction/src/main/scala/org/broadinstitute/monster/dap/RedCapClient.scala

```diff
@@ -46,6 +47,11 @@ object RedCapClient {
     val client = new OkHttpClient.Builder()
       .connectTimeout(timeout)
       .readTimeout(timeout)
+      .addInterceptor(new HttpLoggingInterceptor(new Logger() {
+        override def log(s: String): Unit = {
+          logger.info(s)
+        }
+      }).setLevel(Level.BASIC))
```

Comment:
We can adjust this to also dump the headers + body when debugging; we're not doing that by default here to avoid accidentally outputting sensitive data to our prod logs (api tokens, etc.)

Reply:
i'm surprised their levels have custom names like `BASIC` - how does this correspond to the standard log levels, e.g. `DEBUG`/`INFO`/`WARN`/`ERROR`?

Reply:
Yeah, I found this a bit surprising as well, the log levels are defined in [this enum](https://github.com/square/okhttp/blob/master/okhttp-logging-interceptor/src/main/kotlin/okhttp3/logging/HttpLoggingInterceptor.kt#L49). Short answer is: there isn't a 1-1 correspondence, you're more turning on parts of the HTTP request + response to capture.